files/blockinfile.py

```diff
@@ -141,9 +141,12 @@
       {{item.ip}} {{item.name}}
     marker: "# {mark} ANSIBLE MANAGED BLOCK {{item.name}}"
   with_items:
-      - { name: host1, ip: 10.10.1.10 }
```

Comment:
I'm going to say that rewriting the `write_items` section here decreases readability.

Therefore closing this PR